modules/store/testing/src/mock_selector.ts

```diff
@@ -0,0 +1,11 @@
+import { MemoizedSelector, MemoizedSelectorWithProps } from '@ngrx/store';
+
+export interface MockSelector<State, Result>
+  extends MemoizedSelector<State, Result> {
+  setResult: (result?: Result) => void;
```

Comment:
I think we don't need `setResult` as it's available on `MemoizedSelector`

Reply:
We can probably remove these. I wanted a clearer separation that `setResult` would only be available during testing, but it has to be on the signature of the selector to use it.